docs/Gemfile

```
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+gem 'jekyll'
+gem 'github-pages', group: :jekyll_plugins
```

RuboCop (1.76.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `github-pages` should appear before `jekyll`.

(Bundler/OrderedGems)